lib/cdo/hamburger.rb

```diff
@@ -10,18 +10,22 @@ class Hamburger
   HIDE_ALWAYS = "hide-always".freeze
   SHOW_MOBILE = "show-mobile".freeze
 
+  def self.get_divider_visibility(visibility1, visibility2)
```

Comment:
what do `visibility1` and `visibility2` represent here?

Reply:
I can rename them but they represent the visibility about the divider and below. The divider should only be shown if both sections are shown and I had trouble reasoning about these dividers in a more-hard coded way.

Reply:
ahh, makes sense. Yeah, please do rename them to clarify